src/Liquidator.sol

```diff
@@ -448,23 +440,39 @@ contract Liquidator is Owned, ILiquidator {
         uint256 accountValue = IAccount(account).getAccountValue(IAccount(account).baseCurrency());
         if (accountValue != 0) revert Liquidator_AccountValueIsNotZero();
 
+        _endAuction(
+            account,
+            auctionInformation_.originalOwner,
+            auctionInformation_.initiator,
+            auctionInformation_.creditor,
+            auctionInformation_.startDebt
+        );
+    }
+
+    /**
+     * @notice Ends an auction, settles the liquidation and transfers all remaining assets of the Account to the procotol owner.
+     * @param account The account to end the liquidation for.
+     */
+    function _endAuction(address account, address originalOwner, address initiator, address creditor, uint256 startDebt)
+        internal
+    {
         // Stop the auction, this will prevent any possible reentrance attacks.
         auctionInformation[account].inAuction = false;
 
-        uint256 startDebt = auctionInformation_.startDebt;
         address owner_ = owner;
```

Comment:
Not sure if it should be the owner of the Liquidator contract, or the owner of the creditor, but this for a different PR.

Reply:
What could be nice to implement is a type of locker (just a contract) where the remaining assets are sent (locker handled by risk manager). And a public function to swap assets to baseCurrency and transfer it to lending pool (lendingPool and baseCurrency hardcoded in the swap). So lenders wouldn't rely purely on the risk manager to get assets back. I have already that type of contract coded for 1inch if interested. Would at least work for ERC20 tokens.

Reply:
let's keep that on our to-do list for later